### DISCUSSION TITLE
Videos do not rotate

### DISCUSSION THREAD

**Discussion by Shinkoku** [smplayer-dev/smplayer#1098] [Q&A] [answered]
No matter what angle I choose, in Video - Rotate, videos won't rotate at all. 
Do I have to enable some certain settings to enable rotation?
Thanks

**Comment by Raza-0007** [comment, marked as answer]
Check [this post](https://github.com/smplayer-dev/smplayer/discussions/778#discussioncomment-6706558) for answer.

**Comment by Shinkoku** [comment]
Thanks a lot! 
This helped :)
